images/.#cockpit-images.yaml

```diff
@@ -0,0 +1 @@
+sanne@worky.17978:1542204001
```

Comment:
We don't want that bit :)

Reply:
woops, guess I didn't setup my info/exclude on this one :)